lomap/dbmol.py

```diff
@@ -379,8 +391,8 @@ def ecr(mol_i, mol_j):
             
             for atom in mol_j.GetAtoms():
                 total_charge_mol_j += float(atom.GetProp('_TriposPartialCharge'))
-
-            if abs(total_charge_mol_j - total_charge_mol_i) < 1e-3:
+            #sliu 07/17 change to a loose threshold
+            if abs(total_charge_mol_j - total_charge_mol_i) < 1e-2:
```

Comment:
Should the threshold here be an optional argument to this function? I'm fine to having it default to your new behavior, just asking -- will there ever be cases where we need to change this?